views/interlibrary_loan_requests.erb

```diff
@@ -10,14 +10,14 @@
 <table>
   <caption>
     <div class="loan-caption-inner-layout">
-      <p>Showing <strong><%= interlibrary_loan_requests.count %></strong> items</p>
+      <p>Showing <strong><%= interlibrary_loan_requests.count %></strong> <%= interlibrary_loan_requests.count == 1 ? "item" : "items" %></p>
```

Comment:
This should a method in `models/items/items.rb`. maybe:
```
def item_text
  count ==1 ? "item" : "items"
end
```
And then write a corresponding test. (Which is easier to do in the model than for the view)